sci-chemistry/openbabel/openbabel-3.1.1.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+WX_GTK_VER="3.0"
+
+inherit cmake desktop toolchain-funcs wxwidgets
+
+MY_P="${PN}-$(ver_rs 1- -)"
+
+DESCRIPTION="Interconverts file formats used in molecular modeling"
+HOMEPAGE="http://openbabel.org/wiki/Main_Page"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${MY_P}.tar.gz"
+
+S="${WORKDIR}/${PN}-${MY_P}"
+
+# See src/CMakeLists.txt for LIBRARY_VERSION
+SLOT="0/7.0.0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+IUSE="cairo cpu_flags_arm_neon cpu_flags_x86_sse2 cpu_flags_x86_sse4_2 doc openmp test wxwidgets"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-cpp/eigen:3
+	dev-libs/libxml2:2
```

Comment:
If we link against any libraries here and they have subslots, make sure we subscribe to them with := (this applies to all ebuilds).